<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSeqDialog</class>
 <widget class="QDialog" name="AddSeqDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose a sequence</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="dialogGridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="custom_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="seq_text_edit">
        <property name="placeholderText">
         <string>Paste sequence here or select an option below</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QGroupBox" name="group_box">
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>custom_button_box</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>custom_button_box</sender>
   <signal>rejected()</signal>
   <receiver>AddSeqDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>custom_button_box</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>AddSeqDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
